pkg/operation/cloudbotanist/cloudbotanist.go

```diff
@@ -53,6 +54,10 @@ func New(o *operation.Operation, purpose string) (CloudBotanist, error) {
 		return alicloudbotanist.New(o, purpose)
 	case gardenv1beta1.CloudProviderOpenStack:
 		return openstackbotanist.New(o, purpose)
+	case gardenv1beta1.CloudProviderPacket:
+		return packetbotanist.New(o, purpose)
+	case gardenv1beta1.CloudProviderLocal:
```

Comment:
The local provider has been removed - please delete it here

Reply:
Done